doc/mgr/dashboard.rst

```diff
@@ -359,16 +352,14 @@ To use the Object Gateway management functionality of the dashboard, you will
 need to provide the login credentials of a user with the ``system`` flag
 enabled.
 
-If you do not have a user which shall be used for providing those credentials,
-you will also need to create one::
+If you do not have an appropriate RGW user already, you will need to create one::
```

Comment:
My only thoughts here is that the previous sentence indicated what kind of "appropriate RGW" user was required. This does not. Up to you.

Reply:
Yes, which is why this sentence doesn't need to specify.  I've reworded a bit, PTAL